openairclim/calc_cont.py

```diff
@@ -0,0 +1,340 @@
+"""
+Calculates the contrail response.
+
+_author_: Liam Megill
+_email_: liam.megill@dlr.de
+"""
```

Comment:
Maybe mentioning in the header that this version is based on the AirClim framework. Just for my own curiosity, are _author_ and _email_ special variables (because of the underscores)?

Reply:
Yes, that's a good idea, I'll add that. And I think I did the underscore wrong. It's meant to be (outside of the block comment) __author__ = "Liam Megill" etc. I did this out of habit, I think it's probably best to just remove since in the end there will (hopefully) be many contributors!

Reply:
You can add your name in the module, that is fine.

Apart from that we have a central [citation file](https://github.com/dlr-pa/oac/blob/dev/CITATION.cff)